src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,44 @@
+package duke.task;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Event extends Task {
+    private LocalDate at;
+
+<<<<<<< HEAD
+    /**
+     * Constructor for Event.
+     *
+     * @param input event title.
+     * @param at date/time of event.
+     * @throws DateTimeParseException
+     */
+    public Event(String input, String at) throws DateTimeParseException{
+=======
+    public Event(String input, String at) throws DateTimeParseException {
```

Comment:
Should the conflicts be merged before pushing to the master branch?

Reply:
Agreed, conflicts should be resolved before committing and pushing